build-support/docker/centos6/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# Use centos6 for compatibility with glibc 2.12.
+FROM centos:6
+
+# Ensure Pants runs under the 2.7.13 interpreter.
+ENV PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS="['CPython==2.7.13']"
```

Comment:
can always pass this in at runtime vs hardcoding in the image build, fwiw.

Reply:
Mm, yes. Will do that.